src/mail/editor/MailEditor.ts

```diff
@@ -125,6 +125,10 @@ export class MailEditor implements Component<MailEditorAttrs> {
 		const model = a.model
 		this.sendMailModel = model
 		this.templateModel = a.templateModel
+
+		// if we have any CC/BCC recipients, we should show these so, should the user send the mail, they know where it will be going to
+		this.areDetailsExpanded = (model.bccRecipients().length + model.ccRecipients().length) > 0
```

Comment:
We should also change line 118 to just be a declaration

Reply:
Good call! Fixed.